src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -59,6 +59,54 @@
                    Style="{StaticResource DisclaimerStyle}"
                    Visibility="{x:Bind State.Profile.IsBaseLayer}" />
 
+        <Grid Height="184">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="24" />
+                <RowDefinition Height="160" />
+            </Grid.RowDefinitions>
+            <Grid Grid.Row="0"
+                  Margin="8,0,8,0">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="auto" />
+                </Grid.ColumnDefinitions>
+                <Border MinWidth="128"

```

Comment:
```suggestion
                <!-- Tab Preview -->
                <Border MinWidth="128"
```
or some kind of comment. Took me a sec to understand that this whole thing is creating a fake tab.